whitepaper/wlroots.tex

```diff
@@ -92,7 +92,7 @@ \subsection{Backends}\label{high level backends}
 Also available is the \emph{multi} backend, which acts as a container for many
 backends. This is necessary, for example, to use DRM and libinput together. It
 also allows you to add and remove backends at runtime. An example use-case:
-adding an RDP backend at runtime.
+adding an rdp backend at runtime.
```

Comment:
I think this should be capitalized properly.

Reply:
-_-